editoast/src/converters/generate_routes.rs

```diff
@@ -0,0 +1,441 @@
+use crate::schema::{utils::Identifier, *};
+use std::collections::HashMap;
+
+/*
+In order to build all the routes, we must do a graph search.
+This module provides this graph search and can be understood in three different parts
+- part 1: type definitions for nodes and edges
+- part 2: build the graph
+- part 3: compute the routes
+ */
```

Comment:
We can set this comment as a module documentation.

```suggestion
//! In order to build all the routes, we must do a graph search.
//! This module provides this graph search and can be understood in three different parts:
//!
//! - part 1: type definitions for nodes and edges
//! - part 2: build the graph
//! - part 3: compute the routes

use crate::schema::{utils::Identifier, *};
use std::collections::HashMap;
```